runtime/src/iree/tooling/comparison.cc

```diff
@@ -24,10 +24,16 @@ IREE_FLAG(float, expected_f32_threshold, 0.0001f,
           "Threshold under which two f32 values are considered equal.");
 IREE_FLAG(double, expected_f64_threshold, 0.0001,
           "Threshold under which two f64 values are considered equal.");
+IREE_FLAG(bool, use_relative_error, false,
```

Comment:
since this maps to an enum we should make the flag an enum - it can then be string `exact` `absolute` `relative` (or whatever) e.g. `--equality_mode=relative`